test/transport/core.test.js

```diff
@@ -1,643 +0,0 @@
-'use strict'
```

Comment:
why was this deleted?

Reply:
Because it was nested under `core.test.js` in my IDE. I thought that meant it was a generated file.